eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+# v2: Updated API Language behaviour
+
+## Objective
+
+Simplify usage of the product for developers, avoid common mistakes when dealing with multi language installs,
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
```

Comment:
As for your suggestion, @supriyabhargava is something like what @rolandbenedetti wrote more clear?

e.g. updated version:
>
> As an Editor, in Platform UI I want to be able to select another language when managing entities from the repository (e.g. content, content types, user, user group...) so that I can create update or read those entities in different language without having to access different instances of Platform UI.
>
> As an Editor, I want to be able to select the language of Platform UI interface _(e.g. menus, labels, buttons... which are not coming from the repository as opposed to entities mentioned above)_ independently of the current SiteAccess so that I can use the UI in different languages without having to access different instances of Platform UI.
>

As for the first one @rolandbenedetti had I did not intend to imply user would have to select a default language for entities also, I was thinking that would come from config. But it is technically possible it would be taken language from the one user chooses as interface language, and use that as default for entities _(with fallback to main)_. That would mean languages will never be taken from config at all in the UI.

/cc Some input from @dpobel or others in the UI team would be great as well here.